### PR TITLE
Add frontend for the client_api

### PR DESCRIPTION
**Pull request by BoroviyOrest**
Add frontend for the client API

### CODE REVIEW

File: client_frontend/src/components/Login/index.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        <PublicLayout>
+            <Form
+                {...layout}
+                name="normal_login"
+                initialValues={{remember: true}}
+                onFinish={onSubmit}
+            >
+                <Form.Item
+                    name="username"
+                    rules={[{required: true, message: 'Please input your Username!'}]}
```
Comment (DanilaMonets) [inline client_frontend/src/components/Login/index.js:21]: Nice validation!

File: client_frontend/src/components/Registration/index.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+                >
+                    <Input/>
+                </Form.Item>
+                <Form.Item
+                    label="Email"
+                    name="email"
+                    rules={[
+                        {
+                            type: 'email',
+                            message: 'The input is not valid E-mail!',
```
Comment (DanilaMonets) [inline client_frontend/src/components/Registration/index.js:47]: I'm not sure about this message, but we can discuss it later in scope of another tasks

**Review by DanilaMonets — APPROVED** [review]

File: client_frontend/src/components/Registration/index.js (hunk 2 above)
Comment (BoroviyOrest) [reply client_frontend/src/components/Registration/index.js:47]: okay

File: client_frontend/src/components/Login/index.js (hunk 1 above)
Comment (BoroviyOrest) [reply client_frontend/src/components/Login/index.js:21]: thank you, bro!

**Pull request merged**